### PR TITLE
Fractured mpsa

### PR DESCRIPTION
**Pull request by rbe051**
Low pressure stimulation

### CODE REVIEW

File: src/porepy/numerics/parabolic.py [hunk 1]
```diff
@@ -64,30 +65,41 @@ def space_disc(self):
     problem.solve()
     '''
 
-    def __init__(self, gb, physics='transport', **kwargs):
+    def __init__(self, gb, physics='transport',time_step=1.0, end_time=1.0, **kwargs):
```
Comment (rbe051) [inline src/porepy/numerics/parabolic.py]: I added time_step and end_time to the init.
This way the number of cases where you have to overload Parabolic is reduced drastically

File: src/porepy/numerics/parabolic.py [hunk 2]
```diff
@@ -64,30 +65,41 @@ def space_disc(self):
...
         self.physics = physics
-        self._data = dict()
+        self._data = kwargs.get('data', dict())
+        self._time_step = time_step
+        self._end_time = end_time
         self._set_data()
+
         self._solver = self.solver()
-        self._solver.parameters['store_results'] = True
+        self._solver.parameters['store_results'] = False
```
Comment (rbe051) [inline src/porepy/numerics/parabolic.py]: Can we remove this parameter variable?

File: src/porepy/numerics/parabolic.py [hunk 3]
```diff
@@ -103,7 +115,11 @@ def step(self):
     def update(self, t):
         'Update parameters to time t'
         for g, d in self.grid():
-            d['problem'].update(t)
+            d[self.physics + '_data'].update(t)
+
```
Comment (rbe051) [inline src/porepy/numerics/parabolic.py]: The data assigner should now be stored with keyword, e.g., 'flow_data'. I think this is quite consistent with the physics consept used in the parameter class

File: src/porepy/params/bc.py [hunk 4]
```diff
@@ -62,8 +62,15 @@ def __init__(self, g, faces=None, cond=None):
...
+            if faces.dtype==bool:
+                if faces.size != self.num_faces:
+                    raise ValueError('''When giving logical faces, the size of
+                                        array must match number of faces''')
+                faces = np.argwhere(faces)
             if not np.all(np.in1d(faces, bf)):
                 raise ValueError('Give boundary condition only on the boundary')
+            if isinstance(cond, str):
+                cond = [cond] * faces.size
             if faces.size != len(cond):
```
Comment (rbe051) [inline src/porepy/params/bc.py]: first case: bc accepts boolean array
second case: if a string is given, it assumes this is the boundary condition assigned to all given faces.

Ok?

File: src/porepy/numerics/parabolic.py (hunk 1 above)
Comment (keileg) [reply src/porepy/numerics/parabolic.py]: Fair enough.

File: src/porepy/numerics/parabolic.py (hunk 2 above)
Comment (keileg) [reply src/porepy/numerics/parabolic.py]: We don't use it, do we? If not, then clearly.

File: src/porepy/numerics/parabolic.py (hunk 3 above)
Comment (keileg) [reply src/porepy/numerics/parabolic.py]: Good.

File: src/porepy/params/bc.py (hunk 4 above)
Comment (keileg) [reply src/porepy/params/bc.py]: Yes, I've been wanting to do this.

**Comment by keileg** [comment]
There are some issues with meshio that I hope to fix soon, so no merging until then. I will take care of it when we are done.

**Comment by keileg** [comment]
Travis is failing; my guess is it has to do with compute_discharge(). @rbe051 could you have a look?

**Comment by rbe051** [comment]
I think commit 3ba07ae should fix this. I will merge if tests pass

**Pull request merged**